EIC-EA/EIC-community-D8
# [EICNET-1100]feature: Create the library overview with : post content, highlighted items, ...

I implemented also more features to our sources like "prefiltered content type" "default sorting, you can see all of these functions into SourceTypeInterface.php

How to test it : 

- [ ] Go to /admin/structure/context
- [ ] Edit the "Group - Global" and inject the "EIC Search Overview" into the before content region and configure the block like this 

![image](https://user-images.githubusercontent.com/11585507/129361642-be77807f-c1a6-46f9-8359-6126526e94d1.png)

- [ ] Be sure that you have mixed library content (gallery with lot of slides, gallery with single image, documents, video, ...)
- [ ] Go to a detail group
- [ ] You can add content with the "post content" button

PS: only group owner can highlight contents from group and by default you will see highlighted contents at the top
PS2: for now we have sample images for video and document, we still need to decide what we will use

![image](https://user-images.githubusercontent.com/11585507/129361817-2a3d8e88-007e-4e34-b809-5c90f76e638d.png)
